ophyd/positioner.py

```diff
@@ -62,7 +54,7 @@ def low_limit(self):
     def high_limit(self):
         return self.limits[1]
 
-    def move(self, position, wait=True, moved_cb=None, timeout=30.0):
+    def move(self, position, moved_cb=None, timeout=30.0):
```

Comment:
reading down I see that it was.  I think I am ok with this as all of the wait behavior is implemented in the sub-classes, but is one more detail about sub-classing these that needs to be documented and gotten right.